behaviors/wysiwyg.js

```diff
@@ -440,7 +440,7 @@ function initWysiwygBinder(enableKeyboardExtras) {
         // SelectionAPI R'lyeh fhtagn
         range = document.createRange();
         selection = window.getSelection();
-        range.setStart(editable, 1);
+        range.setStartAfter(editable.lastChild);
```

Comment:
Does this have to be an element?  There are other kinds of DOM nodes.

There is code in dom.js to get the lastElement rather than last child, or something like that, somewhere.